<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyNumberKeyboardWid</class>
 <widget class="QWidget" name="MyNumberKeyboardWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: transparent;


</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_space">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>space</string>
     </property>
     <property name="iconSize">
      <size>
       <width>58</width>
       <height>58</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_0">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/new/prefix1/image/number/number20.png</normaloff>:/new/prefix1/image/number/number20.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>58</width>
       <height>58</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/new/prefix1/image/number/number13.png</normaloff>:/new/prefix1/image/number/number13.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>58</width>
       <height>58</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_7">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/new/prefix1/image/number/number17.png</normaloff>:/new/prefix1/image/number/number17.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>58</width>
       <height>58</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_4">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/new/prefix1/image/number/number14.png</normaloff>:/new/prefix1/image/number/number14.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>58</width>
       <height>58</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_5">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/new/prefix1/image/number/number15.png</normaloff>:/new/prefix1/image/number/number15.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>58</width>
       <height>58</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_9">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>9</string>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/new/prefix1/image/number/number19.png</normaloff>:/new/prefix1/image/number/number19.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>58</width>
       <height>58</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton_1">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/new/prefix1/image/number/number11.png</normaloff>:/new/prefix1/image/number/number11.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>58</width>
       <height>58</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>1</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/new/prefix1/image/number/number12.png</normaloff>:/new/prefix1/image/number/number12.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>58</width>
       <height>58</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_6">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/new/prefix1/image/number/number16.png</normaloff>:/new/prefix1/image/number/number16.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>58</width>
       <height>58</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_8">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/new/prefix1/image/number/number18.png</normaloff>:/new/prefix1/image/number/number18.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>58</width>
       <height>58</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_delate">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>delate</string>
     </property>
     <property name="iconSize">
      <size>
       <width>58</width>
       <height>58</height>
      </size>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
